doc/source/user_guide/groupby.rst

```diff
@@ -722,16 +722,45 @@ accepts the special syntax in :meth:`.DataFrameGroupBy.agg` and :meth:`.SeriesGr
   to make it clearer what the arguments are. As usual, the aggregation can
   be a callable or a string alias.
 
+Example:
+-------
+
+Consider the following DataFrame `animals`:
+
 .. ipython:: python
 
-   animals
+    import pandas as pd
 
-   animals.groupby("kind").agg(
-       min_height=pd.NamedAgg(column="height", aggfunc="min"),
-       max_height=pd.NamedAgg(column="height", aggfunc="max"),
-       average_weight=pd.NamedAgg(column="weight", aggfunc="mean"),
-   )
+    animals = pd.DataFrame({
+        'kind': ['cat', 'dog', 'cat', 'dog'],
+        'height': [9.1, 6.0, 9.5, 34.0],
+        'weight': [7.9, 7.5, 9.9, 198.0]
+    })
+
+To demonstrate "named aggregation," let's group the DataFrame by the 'kind' column and apply different aggregations to the 'height' and 'weight' columns:
+
+.. ipython:: python
+
+   result = animals.groupby('kind').agg(
```

Comment:
Can you put this in the groupby docstring instead?

Reply:
Okay

Reply:
Hello, How do I put this in a groupby docstring

Reply:
The example would go here https://github.com/pandas-dev/pandas/blob/fc308235f0ca6242a209c1fe19f71884fb5d50cd/pandas/core/groupby/generic.py#L1339